Src/VimCore/CoreInterfaces.fs

```diff
@@ -2209,6 +2216,12 @@ type VisualSelection =
 
     with
 
+    member x.IsForward =
+        match x with
+        | Character (_, path) -> path.IsSearchPathForward
+        | Line (_, path, _) -> path.IsSearchPathForward
+        | Block _ -> true

```

Comment:
Yes, you are right. I'll fix this.

This member is used to determine the primary selected span only when there is one selected span because visual block mode is intentionally not used for multiple carets. Conceptually visual block mode is very similar to multiple carets (in particular block insert and block append) and meta multiple cursors is conceivable but not likely to be useful.

For an editing workflow, one would typically create a normal single-caret block selection, split it into carets, which puts you in normal mode with a caret on each line and then use either:

- normal mode commands
- visual character mode at each caret

the latter of which is very much like "multi-cursor visual block mode".